dfusion_rust_core/src/models/order.rs

```diff
@@ -8,10 +8,10 @@ use crate::models::{Serializable, RollingHashable, iter_hash};
 #[serde(rename_all = "camelCase")]
 pub struct Order {
     pub account_id: u16,
-    pub sell_token: u8,
     pub buy_token: u8,
-    pub sell_amount: u128,
+    pub sell_token: u8,
```

Comment:
Is this change really required? Usually you can instantiate structs with field names out of order.

Reply:
no i just got annoyed by the order and felt strongly about it in the moment :man_shrugging: